src/163400.c

```diff
@@ -81,7 +81,37 @@ void filemenu_set_cursor_alpha(s32 arg0) {
     filemenu_cursorGoalAlpha2 = arg0;
 }
 
-INCLUDE_ASM(s32, "163400", filemenu_set_cursor_goal_pos);
+void filemenu_set_cursor_goal_pos(s32 windowID, s32 posX, s32 posY) {
+    Window* window = &gWindows[windowID];
+
+    if (D_80249BB0 != 0
+            || get_game_mode() == 0xF
```

Comment:
Do we have a game mode enum? We probably should if we don't

Reply:
No we do not, at least from what I saw

Reply:
nvm Im wrong